group_project_v2/templates/html/stages/team_evaluation.html

```diff
@@ -1,5 +1,5 @@
-<form class="review peer_review" data-review-type="peer_review" action="submit_review" method="POST">
+<div class="review peer_review" data-review-type="peer_review" data-action="submit_review" data-method="POST">
```

Comment:
Why is this needed? Can it be avoided? This is an accessibility headache. I know we don't have to pass accessibility requirements for this block right now, but it's likely going to have to be undone if we ever intend to see this get wider use.